Header header
bool   ready_for_command
bool   error
int32  position_feedback
int32  velocity_feedback
